Branch1

Mars-98 opened the pull request:
I needed to rename an argument in piece class move_loc method, here's the new story ID I forgot to change it, CU-7rc3et. 
I also edited the king class, because I realized that I was only getting the tile of where the current piece is on, when I'm supposed to be getting the tile of each new location.
Lastly, I implemented the Bishop class move_loc.

Mars-98 committed:
gitignore

Mars-98 committed:
test

Mars-98 committed:
test

Mars-98 committed:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	.idea/chessApp.iml
#	.idea/misc.xml

Mars-98 committed:
#76ebhv worked on design for api

Mars-98 committed:
Merge remote-tracking branch 'origin/develop' into develop

Mars-98 committed:
Merge branch 'develop' of https://github.com/Mars-98/chessApp into develop

Mars-98 committed:
Merge branch 'develop' of https://github.com/Mars-98/chessApp into develop

Mars-98 committed:
CU-7cd0a0 edited king class, #7rbgvg edited parent piece class, #7cczyt created bishop class

Mars-98 commented:
Also even though I created another branch, I'm still seeing the commits made a while ago, Idk what's going on

mattshoobs commented:
We can go through git tomorrow, think a lot of your problems are caused by pycharm. Where are you seeing these old commits?

Mars-98 committed:
CU-7cd0a0 edited king class AGAIN

Mars-98 committed:
Delete design_api

Accidentally pushed a file local to another directory, this doesn't need to be in the pull request

Mars-98 committed:
CU-7cd0a0

Mars-98 committed:
Merge remote-tracking branch 'origin/branch1' into branch1

Mars-98 commented:
I see them on every PR I create

Mars-98 committed:
CU-7cd0a0

Mars-98 committed:
CU-7cd0a0

Mars-98 committed:
CU-7cd0a0 and CU-7cczyt

Mars-98 committed:
CU-7cd0a0 forgot 4 more movements in king

Mars-98 committed:
CU-7cd0a0 first move setter

coughenberg commented:
On first glance my guess would be that you made the branch off of the parentPiece or piece branch, so it brings the git commits along with that, because that branch was deleted on remote (github). Since those commit hashes do not appear in the repository, github thinks they are new commits. And it doesn't show the changes, because you are comparing it to develop, where those changes already exist.

That's a rather verbose answer, but I am pretty sure that is what happened. So when you make a new branch, make sure you checkout develop first, then make a new branch from there

Mars-98 committed:
CU-7cd0a0

Mars-98 commented:
Yeah I started doing that after I fully understood what's going on with git and all those shenanigans

coughenberg reviewed (changes requested):
Marking it as changes requested so I know that I am up to date with this branch. Plan is still to approve then merge when unittests are out and this passes all unittests
